src/YieldDisburser.sol

```diff
@@ -59,7 +66,7 @@ contract YieldDisburser is OwnableUpgradeable {
     function castVoteBySignature(uint256[] calldata percentages, bytes calldata signature, address holder) public {
         (uint8 v, bytes32 r, bytes32 s) = abi.decode(signature, (uint8, bytes32, bytes32));
         address signer = ecrecover(keccak256(abi.encodePacked(percentages)), v, r, s);
-        if (signer != holder) revert("Invalid signature");
+        if (signer != holder) revert("Invalid signature"); // TODO not creating custom error in anticipation of #5
```

Comment:
would be better to convert everything to a custom error and then remove them when they aren't relevant. front running changes to a PR not a good habit imo

Reply:
Will amend 👍🏻

Reply:
addressed by bd7cb9f